components/action-mgt/org.wso2.carbon.identity.action.mgt/src/main/java/org/wso2/carbon/identity/action/mgt/constant/ActionMgtSQLConstants.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.action.mgt.constant;
+
+/**
+ * SQL constants for Action management service.
+ */
+public class ActionMgtSQLConstants {
+
+    // Column Names.
+    public static final String ACTION_UUID_COLUMN_NAME = "ACTION_UUID";
+    public static final String ACTION_TYPE_COLUMN_NAME = "ACTION_TYPE";
+    public static final String ACTION_NAME_COLUMN_NAME = "ACTION_NAME";
+    public static final String ACTION_DESCRIPTION_COLUMN_NAME = "ACTION_DESCRIPTION";
+    public static final String ACTION_STATUS_COLUMN_NAME = "ACTION_STATUS";
+    public static final String ACTION_COUNT_COLUMN_NAME = "ACTION_COUNT";
+    public static final String ACTION_ENDPOINT_PROPERTY_NAME_COLUMN_NAME = "ACTION_ENDPOINT_PROPERTY_NAME";
+    public static final String ACTION_ENDPOINT_PROPERTY_VALUE_COLUMN_NAME = "ACTION_ENDPOINT_PROPERTY_VALUE";
```

Comment:
It would be good to group like ActionMgtSQLConstants.Column.ACTION_UUID

and queries can be grouped similarly

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5771/commits/94e90e62016b434a0f37aa2b1f7ff0edc34bb64f